services/frontend/src/components/header/HeaderView.jsx

```diff
@@ -21,12 +21,7 @@ const HeaderView = ({ title, icon, subtitle, extra, backBtn }) => {
       subTitle={subtitle}
       backIcon={
         <Tooltip placement="top" title={intl.formatMessage({ id: "go.back" })}>
-          <Button
-            type="primary"
-            shape="circle"
-            icon={<ArrowLeftOutlined />}
-            aria-label={intl.formatMessage({ id: "go.back" })}
-          />
+          <ArrowLeftOutlined className="circle-icon" />
```

Comment:
I am not able to test how this change renders as I can't log in to the site. I think keycloak is down. I will look into it on Monday